pkg/client/clientcmd/api/types.go

```diff
@@ -83,6 +83,8 @@ type Context struct {
 	AuthInfo string `json:"user"`
 	// Namespace is the default namespace to use on unspecified requests
 	Namespace string `json:"namespace,omitempty"`
+	// NamespacePath is the path to a kubernetes ns file (~/.kubernetes_ns)
```

Comment:
> do you think we could reasonably just nuke namespace-path right now?

My preference would be to have a second pull request so that any ripples from a breaking change are more self-contained.